### PR TITLE
Support for generating APK in Build CI

### PR DESCRIPTION
**Pull request by PatilShreyas**
This PR Includes:
- Added support for generating APK artifact from Build CI.

**Preview:**
Capture

### CODE REVIEW

File: .github/workflows/build-workflow.yml [hunk 1]
```diff
@@ -44,7 +44,14 @@ jobs:
     # 5. Build
     - name: Build with Gradle
       run: ./gradlew build
+
+    # 6. Upload APK Artifact
+    - name: Upload APK
+      uses: actions/upload-artifact@v1
+      with:
+        name: news-app
+        path: app/build/outputs/apk/debug/app-debug.apk
```
Comment (AkshayChordiya) [inline .github/workflows/build-workflow.yml:53]: Is it possible to have a wildcard here instead of absolute path?
Perhaps something like `*.apk`

Comment (PatilShreyas) [reply .github/workflows/build-workflow.yml:53]: Okay. Let me check this. Thanks for suggestion.

Comment (PatilShreyas) [reply .github/workflows/build-workflow.yml:53]: Can I use `app/**/*.apk`? I tested it and it's working.

Comment (AkshayChordiya) [reply .github/workflows/build-workflow.yml:53]: Interesting, maybe `/**/*.apk`?

Comment (PatilShreyas) [reply .github/workflows/build-workflow.yml]: Actually `/` at the beginning means root path of Action's VM. That's why I choose `app/*`. 

See error:

```
  with:
    name: news-app
    path: /**/*.apk

##[error]EACCES: permission denied, scandir '/etc/polkit-1/localauthority'
```

Comment (AkshayChordiya) [reply .github/workflows/build-workflow.yml]: Ah okay. I guess then `app/**/*.apk` seems promising

Comment (PatilShreyas) [reply .github/workflows/build-workflow.yml]: Finally I solved it using `./**/*.apk`. Let me remove unnecessary commits.

**Comment by PatilShreyas** [comment]
@AkshayChordiya I've done the changes as you suggested. LMK if it looks good!

File: .github/workflows/build-workflow.yml (hunk 1 above)
Comment (AkshayChordiya) [reply .github/workflows/build-workflow.yml]: Sweeet!

**Comment by PatilShreyas** [comment]
Thanks!

**Pull request merged**